README.md

```diff
@@ -85,6 +86,23 @@ Check Docker docs [here](https://github.com/moscajs/aedes-cli#docker)
 
 - [Examples](./docs/Examples.md)
 
+## Clusters
+
+Aedes needs on disk dbs like MongoDB and Redis in order to work with clusters. Based on our tests and users reports the best performances/stability are reached when using [aedes-persistence-mongodb] paired with [mqemitter-redis].
+
+Other info:
+
+- [aedes-persistence-redis] have some memory leak. Check [this issue](https://github.com/moscajs/aedes-persistence-redis/issues/73#issuecomment-686306508)
+- [mqemitter-mongodb] could have leaks when used in clusters. Check [this issue](https://github.com/mcollina/mqemitter-mongodb/issues/24)
+- You should assign a __FIXED BROKER ID__ to your Aedes instances:
+
+  ```js
+  var broker = aedes({ id: 'broker' + clusterId })
+  ```
+
+  When a cluster instance dies the new one should take the same id, this is needed because some documents stored in dbs use `brokerId` to identify packets/clients.
+- The repo [aedes-tests](https://github.com/moscajs/aedes-tests) is used to test aedes with clusters and different emitters/persistences. Check its source code to have a starting point on how to work with clusters
```

Comment:
Overall I would add a few notes that if there are bugs/leaks in production scenarios, we encourage people to send pull request and/or reach out to the maintainer for some paid support.